tensorboard/webapp/metrics/views/right_pane/scalar_column_editor/scalar_column_editor_component.scss

```diff
@@ -19,12 +19,22 @@ $_accent: map-get(mat.get-color-config($tb-theme), accent);
 
 :host ::ng-deep .mat-tab-label {
   min-width: 0;
+  padding: 0 30px;
+}
+.editor-controls {
+  // 158px is the height of the Tensorboard header(64px), the tags filter
+  // box(49px), and the ScalarColumnEditor's footer(40px) combined.
+  height: calc(100vh - 153px);
```

Comment:
This does not take into account the case when the number of headers in the controls is to long for the screen height.
<img width="229" alt="Screenshot 2023-03-31 at 2 36 18 PM" src="https://user-images.githubusercontent.com/8672809/229237192-c7df2316-3f5a-45c5-9620-bc4ebbe936a1.png">

I do agree that this component should be unaware of things like "Tensorboard header", "tags filter box". so I followed your first suggestion and made the height 100% - footer height.

Reply:
That makes sense. In that case we need a different approach. In addition to what you've already done:

1. Absolutely position the footer at the bottom:
* position: absolute;
* left: 0; or right: 0;
* bottom: 0;

2. Because it is now absolutely positioned we have to be a bit more explicit for the width:
* width: 100%;
* box-sizing: border-box;

3. Adjust the editor-controls so that they dont overlap with its sibling, the footer. Its not as nice as flex but its acceptable.
* height: calc(100% - 45px);

4. Adjust tab-group so it fills the editor-controls height.
* height: 100%;

In case I missed anything, you can refer to this:
https://github.com/bmd3k/tensorboard/commit/ce750209719ec9c0c2a122acc6ee18904880507a

Reply:
That works great!